src/components/shared/cellRenderers/cellRenderer.scss

```diff
@@ -35,12 +35,20 @@ $iconSize: 14px;
     > .icon-only { display: none !important; }
   }
 
-  .soft-select-link-cell {
+  .soft-select-link {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
     display: inline;
     color: inherit;
     text-decoration: underline;
+    font-family: inherit;
+    padding: 0px;
```

Comment:
Since this is 0 it doesn't need to have a unit and shouldn't according to the style guide.